src/shared/codelens/localLambdaRunner.ts

```diff
@@ -51,7 +51,8 @@ export interface OnDidSamBuildParams {
 const TEMPLATE_RESOURCE_NAME: string = 'awsToolkitSamLocalResource'
 const SAM_LOCAL_PORT_CHECK_RETRY_INTERVAL_MILLIS: number = 125
 const SAM_LOCAL_PORT_CHECK_RETRY_TIMEOUT_MILLIS_DEFAULT: number = 30000
-const MAX_DEBUGGER_ATTEMPTS: number = 10
+const MAX_DEBUGGER_RETRIES_DEFAULT: number = 30
```

Comment:
Do we need this if we have a setting with a default value? Can we enforce a number > 0 on the settings page?

Reply:
I don't think we can enforce that on the settings page. The constant serves as the fallback if configuration fails to produce a value.